airbyte-integrations/connectors/source-drift/source_drift/manifest.yaml

```diff
@@ -33,7 +33,7 @@ definitions:
     type: "DefaultPaginator"
     pagination_strategy:
       type: "CursorPagination"
-      cursor_value: "{{ last_records['next'] }}"
```

Comment:
do we know if this even worked? I'd expect `last_records` to be an array, not an object